sdk/ai/azure-ai-projects/FunctionTools.md

```diff
@@ -0,0 +1,92 @@
+# Function Tool Specifications
+Function tools are the utility allowing developers to provide functions within their code and invoke during streaming or running.   
```

Comment:
Space after chapter title. If you install the Microsoft MD plug in for VS Code is will show squiggly lines indicating a space is needed.

Reply:
Resolved